### PR TITLE
fixed dates from today not showing up correctly

### PR DESCRIPTION
**Pull request by joeyede**

### CODE REVIEW

File: src/views/ProtestPage.js [hunk 1]
```diff
@@ -77,7 +77,15 @@ function useFetchProtest() {
 
 function getFutureDates(dateTimeList) {
   if (dateTimeList?.length) {
-    return dateTimeList.filter((dateTime) => new Date(dateTime.date) >= new Date());
+    return dateTimeList.filter((dateTime) => {
+      const now = new Date();
+      const today = new Date(Date.UTC(now.getUTCFullYear(), now.getUTCMonth(), now.getUTCDate()));
+
+      const demoDate = new Date(dateTime.date);
```
Comment (guytepper) [inline src/views/ProtestPage.js]: Would `protestDate` be clearer for this?

File: src/views/ProtestPage.js [hunk 2]
```diff
@@ -77,7 +77,15 @@ function useFetchProtest() {
...
+      const demoDateNoTime = new Date(Date.UTC(demoDate.getUTCFullYear(), demoDate.getUTCMonth(), demoDate.getUTCDate()));
```
Comment (guytepper) [inline src/views/ProtestPage.js]: What does `NoTime` means?

Comment (joeyede) [reply src/views/ProtestPage.js]: Just date no time

**Comment by guytepper** [comment]
@joeyede can't we just compare today's date to the protest date? Why is it needed create a UTC version for each?

File: src/views/ProtestPage.js (hunk 1 above)
Comment (joeyede) [reply src/views/ProtestPage.js]: yes I will fix that

**Comment by joeyede** [comment]
Need to extract dates to create date object that has no time (actually 00:00:00 time) so that we are only comparing the date element.  Seems safer to do in UTC to avoid timezone difference that is what sample code I found recommended as well.

**Review by guytepper — APPROVED** [review]
@joeyede OK - I'll merge it for now, but in the future I think we can improve this to be neater, maybe by using [date-fns](https://date-fns.org/)?

**Pull request merged**